model_api/cpp/models/src/instance_segmentation.cpp

```diff
@@ -258,8 +328,16 @@ std::unique_ptr<ResultBase> MaskRCNNModel::postprocess(InferenceResult& infResul
         } else {
             obj.mask = raw_cls_mask;
         }
-        result->segmentedObjects.push_back(obj);
-
+        if (confidence > confidence_threshold) {
+            result->segmentedObjects.push_back(obj);
+        }
+        if (has_feature_vector_name) {
+            saliency_maps[obj.labelID - 1].push_back(obj.mask);
```

Comment:
Is that mask resized, or we don't have a flag similar to postprocess_semantic_masks?

Reply:
Resized. There's on top
```Cpp
        if (postprocess_semantic_masks) {
            obj.mask = segm_postprocess(obj, raw_cls_mask, internalData.inputImgHeight, internalData.inputImgWidth);
        }
```